src/network/modules/tokens/types.ts

```diff
@@ -26,13 +26,13 @@ export interface TokenInfo {
 export type TokenExtendedInfo = Map<AttrIndex, any>
 
 export interface TokensInfoParam {
-  address: Address
+  address: string
```

Comment:
Why is this a string now?

Reply:
Good question. Basically we have to go from a string (in the form) to tagged CBOR (for the network). Where we convert what to what is up to us.

We were doing:
```
User -> Form (string) -> App (Address.fromString) -> Library (Address) -> Network (CBOR)
```

And now I'm thinking:
```
User -> Form (string) -> App (still a string) -> Library (still a string) -> Network (CBOR)
```

This allows the front-end app to _not_ import `Address` from ManyJS, which I think is a good thing.

It has definitely made me wonder about how we're naming things and if they make sense. I don't think we're using Address vs Identity vs Symbol consistently.